src/org/scada_lts/config/ScadaConfig.java

```diff
@@ -226,6 +232,8 @@ public Boolean getBoolean(String propertyName, boolean defaultValue) {
 					useACL = Optional.of(result);
 				} else if (HTTP_RETRIVER_DO_NOT_ALLOW_ENABLE_REACTIVATION.equals(propertyName)) {
 					httpRetriverDoNotAllowEnableReactivation = Optional.of(result);
+				} else if (DONT_CREATE_EVETS_FOR_EMAIL_ERROR.equals(propertyName)) {
+					dontCreateEventsForEmailError = Optional.of(result);
```

Comment:
Is it possible to rename this variable , it means, by just put char 'N' :
DONT_CREATE_EVETS_FOR_EMAIL_ERROR - > DONT_CREATE_EVENTS_FOR_EMAIL_ERROR

That's all